apps/neuvector/crds/crds-neuvector.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: neuvector
```

Comment:
```suggestion
  name: neuvector-crds
```

Reply:
Happy to keep the current naming or should it be changed to `neuvector-crds`?

Reply:
[Master branch](https://github.com/hmcts/cnp-flux-config/blob/master/apps/neuvector/patching-crds/crds-neuvector.yaml#L4)